OWNERS

```diff
@@ -6,9 +6,11 @@ approvers:
   - fridex
   - goern
   - kpostoffice
-  - mayacostantini
   - sesheta
+
 reviewers:
   - codificat
   - harshad16
-  - mayacostantini
+
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 14: did not find expected key.